vignettes/benchmarks.Rmd

```diff
@@ -26,6 +26,12 @@ The odbc package is often much faster than the existing
 [RODBC](https://cran.r-project.org/package=RODBC) and DBI compatible
 [RODBCDBI](https://cran.r-project.org/package=RODBCDBI) packages. We'll benchmark writing and reading data from the [nycflights13](https://github.com/tidyverse/nycflights13) package using the three packages.
 
+```{r, include = FALSE}
+# The CRAN version of RODBC uses iODBC, so to use unixODBC we need to 
+# recompile it from source, specifying the odbc manager explicitly:
+# install.packages("RODBC", type = "source", INSTALL_opts="--configure-args='--with-odbc-manager=odbc'")
```

Comment:
See `vignette("develop")` for more details?

Reply:
[53b752c](https://github.com/r-dbi/odbc/pull/670/commits/53b752ca6bfd09bf7b9b30e78f817111a2caa58f).